administration/src/util/translations/de.json

```
@@ -606,5 +606,9 @@
     "referenceNrInvalidLengthError": "Das Aktenzeichen muss eine Länge zwischen {{KoblenzReferenceNumberMinLength}} und {{KoblenzReferenceNumberMaxLength}} haben.",
     "referenceNrLabel": "Aktenzeichen",
     "startDayLabel": "Startdatum"
+  },
+  "shared": {
+    "Accordion-showLess": "Weniger anzeigen",
```

🙃 i would like to keep the camel case conventions for namespaces
```suggestion
    "accordionShowLess": "Weniger anzeigen",
```

Same for showMore